### PR TITLE
Allow generate seed

### PR DESCRIPTION
**Pull request by Sciumo**
See issue https://github.com/Lightning-AI/lit-parrot/issues/54
Allowing seed setting can change generate output.

### CODE REVIEW

File: generate.py [hunk 1]
```diff
@@ -70,30 +72,32 @@ def generate(
...
+    prompt: str = DEFAULT_PROMPT,
     *,
     num_samples: int = 1,
     max_new_tokens: int = 50,
     top_k: int = 200,
     temperature: float = 0.8,
-    checkpoint_dir: Path = Path(f"checkpoints/stabilityai/stablelm-base-alpha-3b"),
+    checkpoint_dir: Path = Path(DEFAULT_CP),
     quantize: Optional[str] = None,
+    seed: Optional[int] = 1234
```
Comment (carmocca) [inline generate.py:83]: Can you rebase this PR so that you only add the `seed` argument? Thank you!

File: generate.py
Comment (carmocca) [inline generate.py:1]: chat.py should get this change too

File: generate.py (hunk 1 above)
Comment (Sciumo) [reply generate.py:83]: sure.

File: generate.py
Comment (Sciumo) [reply generate.py:1]: Okay.

**Pull request closed without merging**